equipmanager.lic

```diff
@@ -42,6 +42,7 @@ class EquipmentManager
 
   def wear_items(items_list)
     items_list.each { |item| wear_item?(item) }
+    fput("sort auto head")
```

Comment:
Yes, I mistook wear_item in wear_equipment_set's as wear_items. After looking more, I think this is a good place because it won't be called repeatedly, just the once.